.gitpod.yml

```diff
@@ -56,3 +56,5 @@ vscode:
     - timonwong.shellcheck
     - vscjava.vscode-java-pack
     - fwcd.kotlin
+    - dbaeumer.vscode-eslint
+    - esbenp.prettier-vscode
```

Comment:
@filiptronicek Would you happen to know if any of these extensions are already installed by default in VS Code?

Reply:
@jankeromnes neither of these are bundled with VS Code by default [[1](https://github.com/gitpod-io/openvscode-server/blob/6cab367ef24e7f0ea0f1600b237519a56b75dcb5/product.json#L678-L739)] (not even in Microsoft-distributed VS Code), but `dbaeumer.vscode-eslint` is recommended for users when they first open JS-like files [[1](https://github.com/gitpod-io/openvscode-server/blob/6cab367ef24e7f0ea0f1600b237519a56b75dcb5/product.json#L110)]

Reply:
Many thanks! Good as is then 😁 👍